news/6543.bugfix

```diff
@@ -0,0 +1 @@
+If possible, the version info for glibc is extracted from ``os`` instead of ``ctypes`` (which may be missing).
```

Comment:
Use the imperative tone for this (e.g. "Try getting the glibc version info using the `os` module before importing `ctypes` ...": https://pip.pypa.io/en/stable/development/contributing/#contents-of-a-news-entry